operator/cmd/mesh/testdata/manifest-generate/input/ingressgateway_k8s_settings.yaml

```diff
@@ -11,9 +11,5 @@ spec:
       k8s:
         service:
           externalTrafficPolicy: Local
-          ports:
```

Comment:
Can we have a test with ports and without ports? there is some issue somewhere around how the ports override is inconsistent

Reply:
The merge should use merge key name in this case. We should have a test with some strategic merge example.